ArmTemplates/APIM/apim-product.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "addDevelopers": {
+            "type": "bool",
+            "defaultValue":true,
+            "metadata": {
+                "description": "Adds the built in Developers group to the Access Control list of the Product"
+            }
+        },
+        "allowAnonymousAccess": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Adds the built in Guests group to the Access Control list.  This allows unauthenticated access to the APIs in this Product."
+            }
+        },
+        "apimServiceName": {
```

Comment:
Think the apimServiceName should be at the top of the parameters list, not after the optional fields

Reply:
Re-ordered params